scripts/building/dev_requirements.txt

```diff
@@ -0,0 +1,11 @@
+setuptools
```

Comment:
what's the difference with this file and the one under /src/promptflow?

Reply:
this is equivalent to .condarc used during conda initialization.
when init conda, we use the .condarc in the same folder
when init python, we need to use this dev_requirements.

both of these files just enable basic python build environment and some tools needed by generate_connection_json.

Not related to /src/promptflow.